tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
+- `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment>`
 
-At the time of writing, the pipeline runs the tests against the Dev environment, and runs all tests. You can edit the pipeline if you wish to do something different.
+- If you have a slow machine or are testing against a slow environment it is advisable to run the tests with the following options: `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment> --timeout 20 --implicit-wait 20`
 
 
-## Authentication
+Below you will find a set of available CLI options. This is not a complete list of every option that we have available as the intention of this document is to get you started with UI tests & not overwhelm you with the numerous options available.
 
-To run the admin tests, the `run_tests.py` script uses `scripts/get_auth_tokens.py`. The `get_identity_info` function logs in as a user and then returns the relevant local storage and cookies for the authenticated user. This is done so that if an authenticated user is required, a test run only needs to log in once rather than once for each test suite.
+Available CLI options: 
 
-After the user has been logged in by the run_tests.py script, the local storage and cookie data for the authenticated user is saved in the `IDENTITY_LOCAL_STORAGE.txt` and `IDENTITY_COOKIE.txt` files. This is done so that if you're running the tests locally, you don't need to authenticate every time you rerun the tests, as the run_tests.py script will use the data they contain if they exist.
+- `-b` - browser to use (we currently only use Chrome)
 
+- `-i` - interpreter to use. We use robot when testing on our own machines and pabot when testing in CI
 
-## Webdriver
-The `webdriver` directory holds `chromedriver`, used by selenium to interact with the browser. If `chromedriver` isn't present in this directory, it is automatically downloaded when the tests are run. You can explicitly download the `chromedriver` version of your choice with `--chromedriver <version>`. Alternatively, you can manually place the `chromedriver` of your choice into the `webdriver` directory. 
+- `--processes` - This option tells pabot how many processes it can use. We stick to 4 in CI
 
-NOTE: The `run_tests.py` script only downloads `chromedriver` if it doesn't already exist. If you wish the run script to download a different version, you'll first have to delete `chromedriver` from the `webdriver` directory.
+- `-e` - This option dictates what environment to test against. You will need a corresponding `.env` file to match the environment you're testing against (i.e. a `.env.dev` if you are testing against development etc.)
```

Comment:
Might be worth saying that the script --help text will show you the available options.  I *think* it does anyway?